include/RED4ext/Callback.hpp

```diff
@@ -344,17 +344,22 @@ class FlexCallback<R (*)(Args...), InlineSize>
         handler = nullptr;
     }
 
+    inline Memory::IAllocator* GetAllocator()
+    {
+        return reinterpret_cast<Memory::IAllocator*>(&allocator);
+    }
+
     void InitializeBuffer(uint32_t aSize)
     {
         if (aSize > InlineSize)
         {
             if (!allocator)
             {
-                allocator = AllocatorType::Get();
+                std::memcpy(&allocator, AllocatorType::Get(), sizeof(uint64_t));
```

Comment:
```suggestion
                std::memcpy(&allocator, AllocatorType::Get(), sizeof(Memory::IAllocator));
```
This is the vtable for allocator.